js/src/product-feed/submission-success-guide/enhanced-conversion/footer.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import AppButton from '.~/components/app-button';
+import CTA from '.~/components/enhanced-conversion-tracking-settings/cta';
+
+const Footer = ( { handleGuideFinish } ) => {
```

Comment:
In naming convention, a callback prop is usually named onSomethingClick instead of starting with a verb.

In addition, if `handleGuideFinish` is a required prop, the following call `handleGuideFinish?.();` should not need the optional chaining operator; if not, setting its fallback to lodash's `noop` is also a good practice since this codebase doesn't have any `fn?.()` use before.

Reply:
Sure! prop renamed. Let me know if that looks good to you @eason9487

Reply:
Thanks for the recommendation! Updated!